llvm/lib/Bitcode/Reader/MetadataLoader.cpp

```diff
@@ -703,13 +727,41 @@ class MetadataLoader::MetadataLoaderImpl {
     return Error::success();
   }
 
-  void upgradeDebugInfo(bool ModuleLevel) {
+  /// Specifies which kind of debug info upgrade should be performed.
```

Comment:
Maybe there could be some more details in this comment about why there are different kinds of debug info upgrade?